multiraft/multiraft.go

```diff
@@ -348,13 +345,12 @@ func (m *MultiRaft) ChangeGroupMembership(groupID uint64, commandID string,
 		groupID:   groupID,
 		commandID: commandID,
 		fn: func() {
-			err := m.multiNode.ProposeConfChange(context.Background(), uint64(groupID),
+			if err := m.multiNode.ProposeConfChange(context.Background(), uint64(groupID),
 				raftpb.ConfChange{
 					Type:    changeType,
 					NodeID:  uint64(nodeID),
 					Context: encodeCommand(commandID, payload),
-				})
-			if err != nil {
+				}); err != nil {
```

Comment:
I'm leaving it as is. Perhaps we shouldn't be using the if err := ...; err != nil { idiom when it occupies more than a line.